test/REGRESSIONS

```diff
@@ -497,7 +485,7 @@ gbt is working on getting rid of this
 
 signal 11 (first seen 2014-03-02)
 ---------------------------------
-[Error matching compiler output for release/examples/benchmarks/shootout/meteor-fast]
+[Error matching program output for release/examples/benchmarks/shootout/meteor-fast]
```

Comment:
The program output failure was first seen 2014-03-02, looks like.  It's possible that someone changed it to compiler failure once that started occurring as well.